### ISSUE TITLE
feat: Supports injecting scripts into the client

### ISSUE THREAD

**Issue by wll8** [coderaiser/cloudcmd#379]
Supports injecting scripts into the client, which is convenient for adjusting page content, customizing functions...

E.g:

Now there is no way to clearly identify the text, if the official does not provide skin or configuration, but provides the function of injecting code.

![image](https://user-images.githubusercontent.com/28223260/174254055-623179ac-a100-4c51-a2d8-c813669a29eb.png)

This will let me solve this problem very quickly.

I tried to query the documentation and code of jq-console and console-io, but there is no customizable solution in cloudcmd.

**Comment by coderaiser** [comment]
Try [User Menu](http://cloudcmd.io/#user-menu)

**Comment by wll8** [comment]
If I want to modify the styles on the page, do I think of the [User Menu]?

**Comment by coderaiser** [comment]
You can load any styles you want with help of User Menu, and then load Console.

**Comment by wll8** [comment]
Is there a way to have the script execute on the page without triggering the user menu?

**Comment by coderaiser** [comment]
There was such way as [plugins](https://github.com/coderaiser/cloudcmd/tree/v12.0.0#using-as-middleware) but support was dropped inf favor of User Menu.

I suggest you to start from User Menu, and then if you will have ideas how to make it in another way we can try :).

**Comment by wll8** [comment]
thanks. I would try the plugin way first

**Comment by coderaiser** [comment]
> thanks. I would try the plugin way first

Plugins not supported since Cloud Commander v11.

**Comment by coderaiser** [comment]
Yes, but why not just use **User Menu** :)? As I said, this already was implemented...

**Comment by coderaiser** [comment]
@wll8  Any progress on this?
